toonz/sources/image/pli/pli_io.cpp

```diff
@@ -473,6 +473,8 @@ void ParsedPli::setMaxThickness(double maxThickness)
 	imp->m_maxThickness = maxThickness;
 };
 
+#define ABS(a)  ((a) < 0 ? (-(a)) : (a))
```

Comment:
`#include <cstdlib>` and use `std::abs()` instead, no point in redefining this as an unoptimised macro.